app/views/my/password.html.erb

```diff
@@ -26,16 +26,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-
 <% html_title l(:label_my_account), l(:button_change_password) %>
 <%= toolbar title: l(:button_change_password) %>
 
 <%= error_messages_for 'user' %>
 <%= styled_form_tag({:action => :change_password},
-                    {:autocomplete => 'off'}) do %>
+                    {:autocomplete => 'off', class: 'change_password large-5 columns -higher'}) do %>
```

Comment:
You are putting layout behaviour into the template itself, which would better be placed in the `change_password` class itself to keep it more readable and easier to change later:

```sass
// in the sass file
.change_password
  @include breakpoint(large)
    @include grid-size(3)
```